src/main/java/practice/StreamPractice.java

```diff
@@ -14,16 +24,27 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(number-> Arrays.stream(number.split(",")))
+                .toList()
+                .stream()
+                .mapToInt(a -> Integer.parseInt(a.trim()))
```

Comment:
there is no point to collect stream to make it stream again
```suggestion
        return numbers.stream()
                .flatMap(number-> Arrays.stream(number.split(",")))
                .mapToInt(a -> Integer.parseInt(a.trim()))
```